packages/quicktype-core/src/language/CSharp.ts

```diff
@@ -189,7 +190,8 @@ export const cSharpOptions = {
         "Object",
         "secondary"
     ),
-    checkRequired: new BooleanOption("check-required", "Fail if required properties are missing", false)
+    checkRequired: new BooleanOption("check-required", "Fail if required properties are missing", false),
+    keepPropertyName: new BooleanOption("keep-property-name", "Keep original field name generate", true),
```

Comment:
Does this change the default behavior? Please make this `false` if the default is currently equivalent to `false`.